### PR TITLE
Documentation design and other updates

### PR DESCRIPTION
**Pull request by sdhiscocks**
- Code change to make distinct Particle Initiator (to make distinct from Gaussian initiators).
- Update copyright/license year
- Update design section
- Add instructions to install direct from GitHub

### CODE REVIEW

File: docs/source/design.rst [hunk 1]
```diff
@@ -1,9 +1,115 @@
...
 
-    Stone Soup Logical Data Flow
\ No newline at end of file
+    Stone Soup Logical Data Flow
+
+The figure above shows the overall flow of data from the various components, showing for example
+how :class:`~.Detection` data can be read directly from a :class:`~.DetectionReader`, or maybe via
+a :class:`~.DetectionSimulator` which uses :class:`~.GroundTruthPath` data. This could also include
+reading data direct from a sensor. Outputs can also be passed into :class:`~.MetricGenerator`, or
+writen to file or database for later analysis. Note that all these components are optional.
```
Comment (oharrald-Dstl) [inline docs/source/design.rst:62]: *written instead of 'writen'.

File: docs/source/design.rst [hunk 2]
```diff
@@ -1,9 +1,115 @@
...
+A key part of Stone Soup is the data types, that allow data to be passed between components. A
+fundamental example of this in Stone Soup is the :class:`~.State` data type. This has a
+:attr:`~.State.state_vector`, and optional :attr:`~.State.timestamp`, which describes the state of
+something at a particular time. One child class of this is the :class:`~.Detection` class, which
+describes a measurement from a sensor at a particular time; or the :class:`~.GaussianState` which
+not only has a :attr:`~.GaussianState.state_vector` (Gaussian mean), but also has a
+:attr:`~.GaussianState.covar` (Gaussian covariance) and such describes the state with uncertainty
+in the form of a multivariate Gaussian distribution.
+
+Stone Soup also employs |duck typing|_, a technique than means that data types are flexible. For
```
Comment (oharrald-Dstl) [inline docs/source/design.rst:96]: '*that means that' instead of 'than means that'

**Comment by etfrogers-dstl** [comment]
Is there a way of easily seeing the docs as built for this version (as opposed to cloning and running it myself)?

**Comment by sdhiscocks** [comment]
> Is there a way of easily seeing the docs as built for this version (as opposed to cloning and running it myself)?

The documentation is built on CircleCI, which you can reach from the artefacts tab. For example on this PR: https://app.circleci.com/pipelines/github/dstl/Stone-Soup/525/workflows/41c17446-95b8-4b25-924f-4857287c0541/jobs/4177/artifacts

**Comment by sdhiscocks** [comment]
@etfrogers-dstl Just to add that whilst this PR is closed, comments are still welcome. We have a number of PR's updating documentation at the moment, and comments welcomed on all as we are trying to improve documentation, with focus on examples, tutorials, demonstrations, etc.

**Pull request merged**